### DISCUSSION TITLE
【图集】霉图鉴赏10

### DISCUSSION THREAD

**Discussion by capacitor1** [share121/inter-knot#1118] [其他]
这回是逆天图doge
![IMG_6517](https://github.com/user-attachments/assets/28b7ee9c-a5e5-4f08-be86-5f322d4623da)
![IMG_6541](https://github.com/user-attachments/assets/678e4df5-d8dc-43b4-b27c-96716d3bbd89)
![IMG_6542](https://github.com/user-attachments/assets/35c0e957-3154-4d85-8395-7009ef9ff3f6)
![IMG_6543](https://github.com/user-attachments/assets/769ea310-7aa8-4519-9a2f-976a04c2ea94)
![IMG_6544](https://github.com/user-attachments/assets/841cb441-3ad3-4dc9-9330-3ec3424ea0bd)
![IMG_6545](https://github.com/user-attachments/assets/df67193b-0167-48f9-9475-76d968a1bfe4)
![IMG_6548](https://github.com/user-attachments/assets/6241461a-83e7-4425-8e6a-12ef9e96dfa5)
![IMG_6549](https://github.com/user-attachments/assets/423d8a78-4205-4682-a64e-a53df57264e7)

**Comment by Erdbronie** [comment]
图吧禁片（悲
